### PR TITLE
Add Swoft

### PR DESCRIPTION
**Pull request by wujunze**
add  awesome-swoft     Modern High performance AOP and Coroutine PHP Framework, base on Swoole

[Insert URL to the list]
- [Swoft](https://github.com/wujunze/awesome-swoft#readme) -  Modern High performance AOP and Coroutine PHP Framework, base on Swoole.

[Explain what this list is about and why it should be included here]
Modern High performance AOP and Coroutine PHP Framework, base on Swoole
AND  It is a high-performance, popular PHP microservice framework It now has 1.8k stars on GitHub

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -250,6 +250,7 @@
 - [Pyramid](https://github.com/uralbash/awesome-pyramid#readme)
 - [Play1 Framework](https://github.com/PerfectCarl/awesome-play1#readme)
 - [CakePHP](https://github.com/friendsofcake/awesome-cakephp#readme) - PHP framework.
+- [Swoft](https://github.com/wujunze/awesome-swoft#readme) -  Modern High performance AOP and Coroutine PHP Framework, base on Swoole.
```
Comment (DanailMinchev) [inline readme.md]: As per https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md 
```
The entry in the Awesome list should:
- Be added at the bottom of the appropriate category.
````
I understand that in **Back-End Development** we have various technologies and you probably wanted to add it in the end of the PHP frameworks, so probably it should be after **Lumen**, but per rules it should be in the end of the section after **Vapor**

**Comment by wujunze** [comment]
Thanks I have updated the PR, please review it again @DanailMinchev

**Comment by DanailMinchev** [comment]
@wujunze I will also suggest once you finish your list to use https://github.com/sindresorhus/awesome-lint tools which is great automatic validation of your list.

I have executed the command for you and this is the current report:
```
$ npm install --global awesome-lint
$ awesome-lint README.md

  README.md:111:1
  ✖    1:1    Git repository must be at least 30 days old                    awesome/git-repo-age
  ✖    1:1    Missing file contributing.md                                   awesome/contributing
  ✖    1:1    Missing or invalid Table of Contents                           awesome/toc
  ✖    1:18   Invalid badge source                                           awesome/badge
  ✖    1:174  Readme must not contain CI badge                               awesome/no-ci-badge
  ✖    9:3    Invalid list item link URL                                     awesome/list-item
  ✖   10:3    Invalid list item link URL                                     awesome/list-item
  ✖   11:3    Invalid list item link URL                                     awesome/list-item
  ✖   12:3    Invalid list item link URL                                     awesome/list-item
  ✖   13:3    Invalid list item link URL                                     awesome/list-item
  ✖   14:3    Invalid list item link URL                                     awesome/list-item
  ✖   15:3    Invalid list item link URL                                     awesome/list-item
  ✖   16:3    Invalid list item link URL                                     awesome/list-item
  ✖   17:3    Invalid list item link URL                                     awesome/list-item
  ✖   18:3    Invalid list item link URL                                     awesome/list-item
  ✖   19:3    Invalid list item link URL                                     awesome/list-item
  ✖   20:3    Invalid list item link URL                                     awesome/list-item
  ✖   21:3    Invalid list item link URL                                     awesome/list-item
  ✖   22:3    Invalid list item link URL                                     awesome/list-item
  ✖   23:3    Invalid list item link URL                                     awesome/list-item
  ✖   24:3    Invalid list item link URL                                     awesome/list-item
  ✖   25:3    Invalid list item link URL                                     awesome/list-item
  ✖   26:3    Invalid list item link URL                                     awesome/list-item
  ✖   27:3    Invalid list item link URL                                     awesome/list-item
  ✖   31:1    Marker style should be -                                       unordered-list-marker-style
  ✖   36:1    Marker style should be -                                       unordered-list-marker-style
  ✖   43:22   Remove empty section: "Developer Tools"                        no-empty-sections
  ✖   47:26   Remove empty section: "Testing & Debugging"                    no-empty-sections
  ✖   50:37   Remove empty section: "Authentication & Authorization"         no-empty-sections
  ✖   54:16   Remove empty section: "Utilities"                              no-empty-sections
  ✖   58:34   Remove empty section: "Media & Document Management"            no-empty-sections
  ✖   61:7    Text "Javascript" should be written as "JavaScript"            awesome/spell-check
  ✖   61:34   Remove empty section: "Integration with Javascript"            no-empty-sections
  ✖   65:44   Remove empty section: "Databases, ORMs, Migrations & Seeding"  no-empty-sections
  ✖   68:13   Remove empty section: "Search"                                 no-empty-sections
  ✖   71:11   Remove empty section: "APIs"                                   no-empty-sections
  ✖   77:1    Marker style should be -                                       unordered-list-marker-style
  ✖   77:58   List item description must end with proper punctuation         awesome/list-item
  ✖   79:15   Remove empty section: "Payments"                               no-empty-sections
  ✖   83:1    Marker style should be -                                       unordered-list-marker-style
  ✖   83:73   List item description must end with proper punctuation         awesome/list-item
  ✖   85:17   Remove empty section: "Monitoring"                             no-empty-sections
  ✖   89:19   Remove empty section: "Localization"                           no-empty-sections
  ✖   97:1    Marker style should be -                                       unordered-list-marker-style
  ✖   97:62   List item description must end with proper punctuation         awesome/list-item
  ✖  100:23   Remove empty section: "Application Hosting"                    no-empty-sections
  ✖  104:26   Remove empty section: "Application Deployment"                 no-empty-sections
  ✖  108:17   Remove empty section: "Code Snippets"                          no-empty-sections
  ✖  111:1    Don’t add a trailing . to headings                             no-heading-punctuation
  ✖  113:1    Marker style should be -                                       unordered-list-marker-style
  ✖  115:10   Remove empty section: "Videos"                                 no-empty-sections
  ✖  127:9    Remove empty section: "Books"                                  no-empty-sections
  ✖  131:20   Remove empty section: "Starter Projects"                       no-empty-sections
  ✖  135:27   Remove empty section: "Codebases for Reference"                no-empty-sections
  ✖  138:30   Remove empty section: "Content Management Systems"             no-empty-sections
  ✖  142:12   Remove empty section: "Podcasts"                               no-empty-sections
  ✖  150:24   Remove empty section: "Local User Groups"                      no-empty-sections
  ✖  158:8    Remove empty section: "Jobs"                                   no-empty-sections
  ✖  162:28   Remove empty section: "Hosted Development Tools"               no-empty-sections
  ✖  165:17   Remove empty section: "Miscellaneous"                          no-empty-sections
  ✖  174:1    Marker style should be -                                       unordered-list-marker-style
  ✖  175:1    Marker style should be -                                       unordered-list-marker-style
  ✖  176:1    Marker style should be -                                       unordered-list-marker-style
  ✖  177:1    Marker style should be -                                       unordered-list-marker-style
  ✖  178:1    Marker style should be -                                       unordered-list-marker-style
  ✖  182:2    License image must be SVG                                      awesome/license

  66 errors
```

Thanks

**Comment by wujunze** [comment]
Thanks

File: readme.md [hunk 2]
```diff
@@ -267,6 +267,7 @@
 - [Vert.x](https://github.com/vert-x3/vertx-awesome#readme) - Toolkit for building reactive apps on the JVM.
 - [Terraform](https://github.com/shuaibiyy/awesome-terraform#readme) - Tool for building, changing, and versioning infrastructure.
 - [Vapor](https://github.com/Cellane/awesome-vapor#readme) - Server-side development in Swift.
+- [Swoft](https://github.com/wujunze/awesome-swoft#readme) -  Modern High performance AOP and Coroutine PHP Framework, base on Swoole.
```
Comment (sindresorhus) [inline readme.md]: > Modern high-performance AOP and coroutine PHP framework based on Swoole.

**Comment by DanailMinchev** [comment]
Also, you are missing `awesome` GitHub topic as per rules in https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md#requirements-for-your-awesome-list:
```
The repo should have awesome-list & awesome as GitHub topics. I encourage you to add more relevant topics.
```

**Comment by sindresorhus** [comment]
Ping @wujunze

**Comment by uraimo** [comment]
Also @wujunze, please not that now there is a new rule that asks to remove travis/circleci badges since they are not really all that useful for your readers.
(And I agree, empty categories should be removed, you could add them in the future, if needed)

**Comment by wujunze** [comment]
OK  Thanks

**Pull request closed without merging**